src/components/WelcomeModal/WelcomeModal.tsx

```diff
@@ -75,112 +75,66 @@ const WelcomeModal = () => {
     } catch (e) {}
   };
   return (
-    <Modal
-      centered
-      footer={
-        <Button
-          variant={Variant.Primary}
-          onClick={handleWelcomeClosed}
-          data-cy="close-welcome-modal"
+    <ClassNames>
+      {({ css }) => (
+        <Modal
```

Comment:
Perhaps we could use the large size modal here? IMO the videos right now are too small to be useful.